examples/darwin-framework-tool/commands/common/CHIPCommandBridge.mm

```diff
@@ -57,6 +58,48 @@
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR CHIPCommandBridge::GetPAACertsFromFolder(NSArray<NSData *> ** paaCertsResult)
+{
+    NSMutableArray * paaCerts = [[NSMutableArray alloc] init];
+
+    if (!mPaaTrustStorePath.HasValue()) {
+        char * const trust_store_path = getenv(kTrustStorePathVariable);
+        if (trust_store_path != nullptr) {
+            mPaaTrustStorePath.SetValue(trust_store_path);
+        }
+    }
+    if (mPaaTrustStorePath.HasValue()) {
+        NSError * error;
+        NSString * paaStorePath = [NSString stringWithCString:mPaaTrustStorePath.Value() encoding:NSASCIIStringEncoding];
```

Comment:
Followup: Should this be UTF-8 instead of ASCII?

Reply:
Updated to use UTF Encoding.